uberenv.py

```diff
@@ -770,6 +776,17 @@ def patch(self):
                 print("[copying patched packages from {0}]".format(_src_glob))
                 sexe("cp -Rf {0} {1}".format(_src_glob, dest_spack_pkgs))
 
+        # Update spack's config.yaml if clingo was requested
+        if self.use_clingo:
+            import yaml
+            with open(pjoin(spack_etc_defaults_dir, "config.yaml"), 'r') as conf_file:
```

Comment:
Good that this import is deferred b/c pyyaml is not a part of all standard python installs.